website/docs/cloud-docs/api-docs/agents.mdx

```diff
@@ -67,6 +67,7 @@ This endpoint supports pagination [with standard URL query parameters](/cloud-do
 | `sort`         | **Optional.** Allows sorting the returned agents pools. Valid values are `"name"` and `"created-at"`. Prepending a hyphen to the sort parameter will reverse the order (e.g. `"-name"`). |
 | `page[number]` | **Optional.** If omitted, the endpoint will return the first page.                                                                                                                       |
 | `page[size]`   | **Optional.** If omitted, the endpoint will return 20 agent pools per page.                                                                                                              |
+| `filter[allowed_workspaces][name]` | **Optional.** Allows filtering of agent pools based on allowed workspace name. Available in Terraform Enterprise v202207-1 and later.                                |                                                                             |
```

Comment:
Can we explain what allowed workspaces actually means? Do we mean "based on the workspaces that have permission to use the agent pool"?  And can we only have one workspace name? Like is this "for a specific workspace that is configured to use the agent pool"?